ios/FluentUI/ResizingHandleView/ResizingHandleTokens.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Design token set for the `ResizingHandleView` control.
+open class ResizingHandleTokens: ControlTokens {
+    /// Defines the color of the mark of the `ResizingHandle`.
+    open var markColor: DynamicColor {
```

Comment:
Would the background color also be a token for this control?

Reply:
I don't think so, looks like the background of this view is just set to clear, so it gets the background from whatever it's above.

Reply:
Ah. Looks like the other controls do set the background color, so it isn't always clear. I think in that case moving it in the tokens could make sense.